direct/core/src/main/java/cz/o2/proxima/direct/view/LocalCachedPartitionedView.java

```diff
@@ -204,6 +214,21 @@ public boolean onError(Throwable error) {
     }
   }
 
+  @VisibleForTesting
+  long getCurrentTimeMillis() {
```

Comment:
nit: Maybe we can replace this by `Clock` instance passed via constructor, but this is OK as well